sdk/tests/client/node_api.rs

```diff
@@ -343,27 +265,18 @@ async fn test_mqtt() {
     let (tx, mut rx) = tokio::sync::mpsc::channel(BUFFER_SIZE);
 
     client
-        .subscribe(
-            [
-                Topic::new("milestone-info/latest").unwrap(),
-                Topic::new("blocks").unwrap(),
-            ],
-            move |evt| {
-                match &evt.payload {
-                    MqttPayload::Block(_) => {
-                        assert_eq!(evt.topic, "blocks");
-                    }
-                    MqttPayload::Json(_) => {
-                        assert_eq!(evt.topic, "milestone-info/latest");
-                    }
-                    _ => panic!("unexpected mqtt payload type: {:?}", evt),
-                }
-                match tx.try_send(()) {
-                    Ok(_) | Err(TrySendError::Full(_)) => (),
-                    e => e.unwrap(),
+        .subscribe([Topic::new("blocks").unwrap()], move |evt| {
+            match &evt.payload {
+                MqttPayload::Block(_) => {
+                    assert_eq!(evt.topic, "blocks");
                 }
-            },
-        )
+                MqttPayload::Json(_) => {}
```

Comment:
we should panic here like before: `_ => panic!("unexpected mqtt payload type: {:?}", evt),` as we are defenitely not eexpecting json whhen only requesting block right?

Reply:
Done